packages/manager/src/features/CloudPulse/Widget/CloudPulseWidget.tsx

```diff
@@ -331,9 +329,11 @@ export const CloudPulseWidget = (props: CloudPulseWidgetProperties) => {
             areas={areas}
             ariaLabel={ariaLabel ? ariaLabel : ''}
             data={data}
+            dotRadius={1.5}
```

Comment:
sorry, I take back my previous comment, is 1.5 still okay

Reply:
yes for graph with lots of plots, 1.5 was looking much cleaner compared to 2.

Reply:
I agree 1.5 looks better when we analyzed.